src/library/utils/auth.js

```diff
@@ -22,6 +22,7 @@ export const signUp = (email, password) => {
       .catch(error => {
         if (error.code === 'auth/email-already-in-use') {
           console.log('That email address is already in use!');
```

Comment:
Remember that GitHub should not show debugging code like console.log(). If you need to place some text to the end user, place an Alert.